### DISCUSSION TITLE
Nvidia + FFMPEG working?

### DISCUSSION THREAD

**Discussion by supajason** [blakeblackshear/frigate#2028] [General]
Hi,

I hope someone can help as I'm stuck trying to get hardware decoding with my gpu.

I've followed this [link](https://github.com/blakeblackshear/frigate/issues/1847#issuecomment-932076731) and managed to get it pretty much working however when I run 'ffmpeg -decoders | grep cuvid' I don't get any Nvidia CUVID like the [docs](https://docs.frigate.video/configuration/nvdec) show

 V..... h264_cuvid            (codec h264)
 V..... hevc_cuvid            (codec hevc)
 V..... mjpeg_cuvid           (codec mjpeg)
 V..... mpeg1_cuvid           (codec mpeg1video)
 V..... mpeg2_cuvid           (codec mpeg2video)
 V..... mpeg4_cuvid           (codec mpeg4)
 V..... vc1_cuvid             (codec vc1)
 V..... vp8_cuvid             (codec vp8)
 V..... vp9_cuvid             (codec vp9)

is this right I do get output from runnng nvidia-smi?

thanks

**Comment by supajason** [comment]
Finally figured it out..my GPU isn't supported hence why not Nvidia decoders were on the above output.

**Reply by ferbulous** [reply]
Is there a list for supported gpu?
